typescript-circleci/.circleci/config.yml

```diff
@@ -0,0 +1,28 @@
+version: 2
+jobs:
+  build:
+    machine: true
+    working_directory: ~/repo
+    steps:
+      - checkout
+      - run:
+          name: Start Prisma Server
+          command: |
+            set +e             
+            curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.11/install.sh | bash
```

Comment:
Is there be an easier way to install NodeJS? Ideally in a single line of code.

Reply:
I tried it now for a few hours but i can't find an easier way.